src/TestCases/Traits/AssertTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace Eonx\TestUtils\TestCases\Traits;
+
+use Eonx\TestUtils\Constraints\ArraySameWithDates;
+use Eonx\TestUtils\TestCases\Exceptions\InvalidParentClassException;
+use Eonx\TestUtils\TestCases\TestCase;
+
+trait AssertTrait
+{
+    /**
+     * AssertTrait constructor.
+     */
+    public function __construct()
+    {
+        if ($this instanceof TestCase === false) {
+            throw new InvalidParentClassException('Trait must be used within an Eonx\\TestUtils\\TestCases class.');
```

Comment:
I think we need to make sure that `AssertTrait` is no used by a class which does not extent TestUtils TestCase? Is there a better way to do this. Also this helps make sure we can use parent's static assertions methods.